concepticondata/conceptlists/Williamson-1973-118.tsv

```diff
@@ -0,0 +1,119 @@
+Williamson-1973-118-31	31	ear	oreille	Ohr	1247	EAR
+Williamson-1973-118-32	32	eat	manger	essen; fressen (animals)	1336	EAT
+Williamson-1973-118-33	33	egg	oeuf	Ei; Hühnerei (hen’s egg, chicken’s egg)	744	EGG
+Williamson-1973-118-34	34	elephant	éléphant	Elefant	1290	ELEPHANT
+Williamson-1973-118-35	35	extinguished, be	s’éteindre	erlöschen (vom Licht); ausgelöscht sein	2209	BECOME EXTINGUISHED
+Williamson-1973-118-36	36	eye	oeil	Auge	1248	EYE
+Williamson-1973-118-37	37	fall	tomber	fallen, stürzen, abstürzen, niederfallen, herunterfallen	1280	FALL
+Williamson-1973-118-38	38	fat: oil	graisse: huile	Fett: Öl (Palmöl)	2551	ORGANIC FAT OR OIL
+Williamson-1973-118-39	39	fear	peur	Furcht, Angst	781	FEAR (FRIGHT)
+Williamson-1973-118-40	40	female	femelle	weiblich	1551	FEMALE
+Williamson-1973-118-41	41	firewood	bois de feu	Brennholz, Feuerholz	10	FIREWOOD
+Williamson-1973-118-42	42	forest	forêt: brousse	Wald, Forst, Urwald:Gebüsch	420	FOREST
+Williamson-1973-118-43	43	fowl (chicken)	volaille	Huhn	265	FOWL
+Williamson-1973-118-44	44	fry: bake in ashes	faire frire: faire cuire	braten: in Asche backen	2146	ROAST OR FRY
+Williamson-1973-118-45	45	give: «dash»	donner: faire cadeau	geben: schenken	1447	GIVE
+Williamson-1973-118-46	46	go: walk	aller: marcher	gehen, fahren: zu Fuss gehen	695	GO
+Williamson-1973-118-47	47	grasshopper	sauterelle	Grashüpfer, Heuschrecke	607	GRASSHOPPER
+Williamson-1973-118-48	48	grow: become strong	croître: grandir	wachsen: stark werden	479	GROW
+Williamson-1973-118-49	49	guineafowl	pintade	Perlhuhn	456	GUINEA FOWL
+Williamson-1973-118-50	50	head	tête	Kopf	1256	HEAD
+Williamson-1973-118-51	51	heart	coeur	Herz	1223	HEART
+Williamson-1973-118-52	52	heavy	lourd (être)	schwer	1210	HEAVY
+Williamson-1973-118-53	53	hoe: cultivate	piocher: cultiver	hacken: anbauen	1541	CULTIVATE
+Williamson-1973-118-54	54	kill	tuer	töten	1417	KILL
+Williamson-1973-118-55	55	leg: foot	jambe: pied	Bein: Fuss	2098	FOOT OR LEG
+Williamson-1973-118-56	56	leopard: «panther»	léopard: «panthère»	Leopard: Panther	1139	LEOPARD
+Williamson-1973-118-57	57	lie down: stretch oneself:: sleep	se coucher: s’allonger:: dormir	liegen, sich hinlegen: sich (aus)strecken :: schlafen		
+Williamson-1973-118-58	58	lick	lécher	lecken	319	LICK
+Williamson-1973-118-59	59	man: husband	homme: mari	Mann: Ehemann, Gatte, Gemahl	2646	HUSBAND OR MALE PERSON
+Williamson-1973-118-60	60	meat	viande	Fleisch	634	MEAT
+Williamson-1973-118-61	61	mould	mouler	abformen, modellieren		
+Williamson-1973-118-62	62	mount	monter	hinaufgehen, besteigen, ersteigen	705	GO UP (ASCEND)
+Williamson-1973-118-63	63	name	nom	Name	1405	NAME
+Williamson-1973-118-64	64	night: day of 24 hours	nuit: jour de 24 heures	Nacht: Tag von 24 Stunden		
+Williamson-1973-118-65	65	oil palm: fan palm:: dum palm	palmier à huile: ?:: ?	Ölpalme: ?::?		
+Williamson-1973-118-66	66	person	personne	Person, Mensch	683	PERSON
+Williamson-1973-118-67	67	pig (domestic): (wild)	cochon: phacochère	Ferkel: Wildschwein [?]	1337	PIG
```

Comment:
Not sure about this one. The three 'source' languages are all somewhat different. Leaning towards unmapping.

Reply:
Hmm although the French and German glosses could be mapped to different concept sets if used separately, here they seem to have meant the distinction between the domestic pig and the wild boar (as shown in the English gloss).

Reply:
I wasn't sure either. When in doubt, I would focus on the English gloss, but is the concept of a wild pig included in PIG? Or are PIG and WILD BOAR on the same level without a common hyperonym? I can unmap it, I just wasn't sure whether there was already a consensus.

Reply:
Seems like PIG denotes a domesticated animal "farmed by humans for its meat", according to the current definition of PIG. Better unmap it then I assume, I don't see any hyperonyms for both.